src/Frontend/Modules/FormBuilder/Widgets/Form.php

```diff
@@ -467,7 +467,11 @@ private function validateForm()
                 $fieldName = 'field' . $field['id'];
 
                 // skip
-                if ($field['type'] == 'submit' || $field['type'] == 'paragraph' || $field['type'] == 'heading') {
+                if ($field['type'] == 'submit' ||
```

Comment:
I don't agree
if you are saying a list outloud you also dont say I have this and, this and, this
you say I have this, and this, and this